ansi/elements.go

```diff
@@ -227,6 +227,13 @@ func (tr *ANSIRenderer) NewElement(node ast.Node, source []byte) Element {
 		u := string(n.URL(source))
 		label := string(n.Label(source))
 		if n.AutoLinkType == ast.AutoLinkEmail && !strings.HasPrefix(strings.ToLower(u), "mailto:") {
+			if tr.context.options.DisableAutoEmailLinks {
+				return Element{
```

Comment:
Hey, thanks for the contribution! This is a feature I know people are interested in. 

One thing to keep in mind is this also strips the link styling of anything that looks like an email. Do we want to keep the ability to style these components with custom styles? Gotta think about it a bit :)

Reply:
😄 Thank you for the review!

For my own use case, I would prefer to just remove the styling for anything that looks like an email. But, I can totally see other people's use case where they want to modify the style instead.

Happy to modify this PR if you think we should not just _disable_ email auto links, but allow them to be disabled or otherwise manipulated through a custom style. But, I might need a little more time to get that sorted out.

Reply:
👋 @bashbunni, just wanted to follow up on this again. Should I try to update this PR to keep the ability to style these components?

Reply:
Hey @picatz I'm going to leave it open for now, I'm currently doing a big refactor of glamour, so will take a look again once that's done.

Reply:
Understood, thank you for the update!